resteasy-jaxrs/src/main/java/org/jboss/resteasy/client/exception/ResteasyBadRequestException.java

```diff
@@ -0,0 +1,103 @@
+package org.jboss.resteasy.client.exception;
+
+import javax.ws.rs.core.Response;
+
+/**
+ * A runtime exception indicating a {@link javax.ws.rs.core.Response.Status#BAD_REQUEST
+ * bad client request}.
+ *
+ * @author Sergey Beryozkin
+ * @author Marek Potociar
+ * @since 2.0
```

Comment:
These should be removed, here and in all the other exception classes being added in this commit

Reply:
Hey @asoldano,

I wanted to ask you about that. I copied these exceptions almost bit by bit from the originals, except for the name change and some fiddling with the response field. That's why I left them in. Do you feel confident about removing them?

Similarly, the original classes have an Oracle EULA. It looks like I removed most but not all of them. What about that?

-Ron